declarations/Twitter.json

```diff
@@ -21,10 +22,7 @@
         ".p21-hidden-content"
```

Comment:
You should also remove background pictures with `.ct01__background` and `.ct17__background`, non necessary illustrations with `.b04__image` and hidden elements on _lg_ breakpoint (to avoid some elements to be displaying two times) with`.twtr-hidden--lg`.